### PR TITLE
Module3

### PR DESCRIPTION
**Pull request by antonpus**

### CODE REVIEW

File: utils/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "streams.js",
+  "scripts": {},
+  "author": "Anton Pus <anton_pus@epam.com>",
+  "license": "ISC",
+  "bin": {
+    "streams-util": "./streams.js"
+  },
+  "devDependencies": {
+    "csv-parser": "^2.1.0"
```
Comment (vinfinit) [inline utils/package.json:13]: package.json is empty. Need 'commander', 'through2' and etc.

File: utils/streams.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    const filterCssFiles = fileNames => fileNames.filter(name => path.extname(name) === '.css');
+    const filterBundleCss = fileNames => fileNames.filter(name => name !== 'bundle.css' && name !== 'default.css');
+    const addDefaultCss = fileNames => {
+        fileNames.push('default.css');
+        return fileNames;
+    }
+
+
+    const getFileContentAndPushIntoStream = fileName => fs.createReadStream(path.join(dirPath, fileName))
+        .on('data', data => cssStream.push(data));
```
Comment (vinfinit) [inline utils/streams.js:144]: Add 'error' handler

**Review by vinfinit — APPROVED** [review]
Great work!

**Comment by antonpus** [comment]
> Great work!

thank you!

**Pull request merged**